src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -0,0 +1,187 @@
+    :returns whether command succeeded
+    """
+    with tempfile.NamedTemporaryFile(mode='w') as tfile:
+        tfile.write('#!/bin/bash -x\n')
+        tfile.write('mkdir -p /root/.ssh\n')
+        tfile.write('chmod 0700 /root/.ssh\n')
+        tfile.write('ssh-keygen -b 2048 -t rsa -f /root/.ssh/id_rsa'
+                    ' -q -N "" <<< y\n')
+        tfile.write(f'HOMEDIR="$(getent -s sss passwd {user}|awk -F:'
+                    f' \'{{print $6}}\')"\n')
+        tfile.write('mkdir -p $HOMEDIR/.ssh\n')
+        tfile.write('chmod 0700 $HOMEDIR/.ssh\n')
+        tfile.write('cat /root/.ssh/id_rsa.pub >> $HOMEDIR/.ssh/'
+                    'authorized_keys\n')
+        tfile.write('chmod 0600 $HOMEDIR/.ssh/authorized_keys\n')
+        tfile.write('echo "StrictHostKeyChecking=no" >> $HOMEDIR/.ssh'
+                    '/config\n')
+        if group:
+            tfile.write(f'chown -R {user}:{group} $HOMEDIR/.ssh\n')
+        else:
+            tfile.write(f'chown -R {user} $HOMEDIR/.ssh\n')
+        tfile.flush()
+        session_multihost.client[0].transport.put_file(
+                tfile.name, '/tmp/ssh_setup.sh')
+    sh_cmd = 'chmod +x /tmp/ssh_setup.sh; /tmp/ssh_setup.sh; echo $?'
+    cmd = session_multihost.client[0].run_command(sh_cmd, raiseonerr=False)
+    return cmd.returncode == 0
+
+
+@pytest.mark.tier1
+@pytest.mark.admultidomain
+@pytest.mark.usefixtures("change_client_hostname")
+class TestADMultiDomain(object):
+
+    @staticmethod
+    def test_0001_bz2013297(multihost, adchildjoin):
+        """
+        :title: IDM-SSSD-TC: ad_provider: forests: disabled root ad domain
+        causes subdomains to be marked offline
+        :id:
+        :setup:
+          1. Configure parent and child domain
+          2. Join client to child domain
+          3. ad_enabled_domains to contain only the child domain
+        :steps:
+          1. Lookup user from child domain
+          2. Lookup user from parent domain
+        :expectedresults:
+          1. User is found
+          2. User is not found
+        :customerscenario: True
+        """
+        adchildjoin(membersw='adcli')
+        ad_domain = multihost.ad[1].domainname
+        ad_child_domain = multihost.ad[1].domainname
+
+        # Configure sssd
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[1])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': ad_child_domain,
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True',
+            'cache_credentials': 'True',
+            'ad_enabled_domains': ad_child_domain
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        client.clear_sssd_cache()
+        # Search for the user in root domain
+        parent_cmd = multihost.client[0].run_command(
+            f"getent passwd user1@%s" % ad_domain,
```

Comment:
We should probably be using f-strings everywhere for consistency instead of the old way in some places.
```
    f'getent passwd user1@{ad_domain}'
```

Reply:
Updated and changed for consistency.